src/main/java/forestry/plugin/DefaultBeeSpecies.java

```diff
@@ -628,6 +631,59 @@ public static void register(IApicultureRegistration apiculture) {
 				})
 				.setAuthority("MysteriousAges");
 
+		// Savanna
+		apiculture.registerSpecies(ForestryBeeSpecies.SAVANNA, GENUS_SAVANNA, SPECIES_SAVANNA, true, new Color(0xfdaa48))
+				.setTemperature(TemperatureType.WARM)
+				.setHumidity(HumidityType.ARID)
+				.addProduct(BEE_COMBS.stack(EnumHoneyComb.PARCHED), 0.20f)
+				.addSpecialty(new ItemStack(Items.RED_SAND), 0.10f)
+				.setGenome(genome -> {
+					genome.set(BeeChromosomes.LIFESPAN, ForestryAlleles.LIFESPAN_SHORT);
+					genome.set(BeeChromosomes.SPEED, ForestryAlleles.SPEED_SLOWER);
+					genome.set(BeeChromosomes.TERRITORY, ForestryAlleles.TERRITORY_LARGE);
+					genome.set(BeeChromosomes.EFFECT, ForestryAlleles.EFFECT_AGGRESSIVE);
+				})
+				.setAuthority("EnderiumSmith");
+
+		// Argil
+		apiculture.registerSpecies(ForestryBeeSpecies.ARGIL, GENUS_SAVANNA, SPECIES_ARGIL, true, new Color(0x96AFD2))
+				.setTemperature(TemperatureType.WARM)
+				.setHumidity(HumidityType.ARID)
+				.addProduct(BEE_COMBS.stack(EnumHoneyComb.KAOLIN), 0.30f)
+				.addSpecialty(new ItemStack(Items.RED_SAND), 0.15f)
+				.setGenome(genome -> {
+					genome.set(BeeChromosomes.LIFESPAN, ForestryAlleles.LIFESPAN_SHORT);
+					genome.set(BeeChromosomes.SPEED, ForestryAlleles.SPEED_SLOW);
+					genome.set(BeeChromosomes.TERRITORY, ForestryAlleles.TERRITORY_LARGE);
+					genome.set(BeeChromosomes.EFFECT, ForestryAlleles.EFFECT_SIFTER);
+				})
+				.addMutations(mutations -> {
+					mutations.add(ForestryBeeSpecies.SAVANNA, ForestryBeeSpecies.DILIGENT, 15)
+							.restrictTemperature(TemperatureType.WARM, TemperatureType.HOT)
+							.restrictHumidity(HumidityType.ARID);
+				})
+				.setAuthority("EnderiumSmith");
+
+		// Pride
+		apiculture.registerSpecies(ForestryBeeSpecies.PRIDE, GENUS_SAVANNA, SPECIES_PRIDE, true, new Color(0x650021))
+				.setTemperature(TemperatureType.WARM)
+				.setHumidity(HumidityType.ARID)
+				.addProduct(BEE_COMBS.stack(EnumHoneyComb.KAOLIN), 0.20f)
+				.addSpecialty(BEE_COMBS.stack(EnumHoneyComb.MELLOW), 0.10f)
+				.setGenome(genome -> {
+					genome.set(BeeChromosomes.LIFESPAN, ForestryAlleles.LIFESPAN_SHORTENED);
+					genome.set(BeeChromosomes.SPEED, ForestryAlleles.SPEED_SLOWER);
+					genome.set(BeeChromosomes.TERRITORY, ForestryAlleles.TERRITORY_LARGER);
+					genome.set(BeeChromosomes.EFFECT, ForestryAlleles.EFFECT_HAKUNA_MATATA);
+				})
+				.addMutations(mutations -> {
+					mutations.add(ForestryBeeSpecies.SAVANNA, ForestryBeeSpecies.ARGIL, 9)
+							//TODO: check is_modified forge tag contents to see if possible substitute
```

Comment:
What does this mean? Tag overrides from other datapacks will work just fine

Reply:
in old versions some unusual biome variants were called "modified" or "mutated" biomes.

Reply:
Ah. Those kinds of biomes no longer exist according to the [Minecraft wiki](https://minecraft.wiki/w/Biome/Before_1.18#Modified_Jungle).